packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -82,12 +82,14 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
        );
 
   static VelocityTracker _defaultBuilder(PointerEvent event) => VelocityTracker.withKind(event.kind);
-  /// Configure the behavior of offsets sent to [onStart].
+
+  /// Configure the behavior of offsets passed to [onStart].
   ///
   /// If set to [DragStartBehavior.start], the [onStart] callback will be called
-  /// at the time and position when this gesture recognizer wins the arena. If
-  /// [DragStartBehavior.down], [onStart] will be called at the time and
-  /// position when a down event was first detected.
+  /// using position where this gesture recognizer wins the arena. If
+  /// [DragStartBehavior.down], [onStart] will be called using the position where
+  /// a down event was first detected. When there are no other gestures to
+  /// compete with this gesture, they are no different.
```

Comment:
Maybe:

... with the position of the pointer at the time this gesture recognizer won the arena. If [DragStartBehavior.down], [onStart] will be called with the position of the first detected down event for the pointer. When there are no other gestures competing with this gesture in the arena, there's no difference in behavior between the two settings.